lib/src/builder/codegen/accessors_generator.dart

```diff
@@ -0,0 +1,503 @@
+  static const String proxiedStateMapName = 'state';
+
+  static const String staticPropKeysName = '${publicGeneratedPrefix}propKeys';
+  static const String staticStateKeysName = '${publicGeneratedPrefix}stateKeys';
+
+  static const String staticPropsName = '${publicGeneratedPrefix}props';
+  static const String staticStateName = '${publicGeneratedPrefix}state';
+
+  static const String staticConsumedPropsName = '${publicGeneratedPrefix}consumedProps';
+
+  String _generateAccessors() {
+    String keyNamespace = getAccessorKeyNamespace(names, member.meta);
+
+    final String proxiedMapName = type.isProps ? proxiedPropsMapName : proxiedStateMapName;
+    final String keyListName = type.isProps ? staticPropKeysName : staticStateKeysName;
+    final String constantListName = type.isProps ? staticPropsName : staticStateName;
+    final String constConstructorName = type.isProps ? 'PropDescriptor' : 'StateDescriptor';
+
+    Map keyConstants = {};
+    Map constants = {};
+
+    StringBuffer output = StringBuffer();
+
+    node.members.whereType<FieldDeclaration>().where((field) => !field.isStatic).forEach((field) {
+      T getConstantAnnotation<T>(AnnotatedNode member, String name, T value) {
+        return member.metadata.any((annotation) => annotation.name?.name == name) ? value : null;
+      }
+
+      final accessorMeta = instantiateAnnotationTyped<annotations.Accessor>(field);
+      final requiredProp = getConstantAnnotation(field, 'requiredProp', annotations.requiredProp);
+      final nullableRequiredProp =
+          getConstantAnnotation(field, 'nullableRequiredProp', annotations.nullableRequiredProp);
+
+      if (accessorMeta?.doNotGenerate == true) {
+        return;
+      }
+
+      field.fields.variables.forEach((variable) {
+        if (variable.initializer != null) {
+          logger.severe(messageWithSpan(
+              'Fields are stubs for generated setters/getters and should not have initializers.\n'
+              'Instead, initialize ${type.isProps ? 'prop values within getDefaultProps()' : 'state values within getInitialState()'}.',
+              span: getSpan(sourceFile, variable)));
+        }
+
+        String accessorName = variable.name.name;
+
+        String individualKeyNamespace = accessorMeta?.keyNamespace ?? keyNamespace;
+        String individualKey = accessorMeta?.key ?? accessorName;
+
+        /// Necessary to work around issue where private static declarations in different classes
+        /// conflict with each other in strong mode.
+        /// TODO remove once https://github.com/dart-lang/sdk/issues/29751 is resolved
+        String staticConstNamespace = node.name.name;
+
+        String keyConstantName =
+            '${privateSourcePrefix}key__${accessorName}__$staticConstNamespace';
+        String keyValue = stringLiteral(individualKeyNamespace + individualKey);
+
+        String constantName = '${privateSourcePrefix}prop__${accessorName}__$staticConstNamespace';
```

Comment:
I guess it should be, I can update this. (FYI this code didn't change in this PR, and was just moved)

Reply:
Actually, I'm going to leave it as-is so that legacy builder output is exactly the same, so we can properly QA this PR. 

We can address this later on, when we remove the `staticConstNamespace`